components/Button.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC } from 'react'
+import { Button as RButton, ButtonProps } from 'rebass/styled-components'
+
+interface IButton {
+  fontSize: number
+  border: number
+  inverted: boolean
+  hollow: boolean
+  variant: string
+}
```

Comment:
This is a bit bikesheddy, but perhaps these interfaces should be called `ButtonProps` instead of `IButton`, since it's an interface for the props of the Button Functional Component, not an interface to the actual functional component itself.